src/shared/lib/tracking.ts

```diff
@@ -0,0 +1,48 @@
+import analytics from 'universal-ga';
+import $ from 'jquery';
+import * as _ from 'lodash';
+
+const trackingCode = 'UA-85438068-1';
```

Comment:
i know this should be in configuration.  we should probably have different Ids for different envs for testing purposes. will work on that